framework/src/main/java/org/checkerframework/framework/type/DefaultTypeHierarchy.java

```diff
@@ -365,7 +365,12 @@ protected boolean isContainedBy(
                 return false;
             }
 
-            return canBeCovariant || checkAndSubtype(outsideWc.getSuperBound(), inside);
+            AnnotatedTypeMirror lowerbound = outsideWc.getSuperBound();
+            if (lowerbound.getKind() == TypeKind.TYPEVAR) {
+                // TODO: determine correct check for (captured) type variables.
```

Comment:
This fixes the crash, but is an ugly work-around.
Anyone have a better suggestion?